main.py

```diff
@@ -155,7 +159,8 @@ def main(args):
             batch_size=args.batch_size,
             save_location=results_dir,
             num_embeddings=default_settings['num_embeddings'],
-            num_samples=default_settings['num_samples']
+            num_samples=default_settings['num_samples'],
+            augment=default_settings['augment']
```

Comment:
Does this mean that the `--aug` CLI parameter does not actually do anything?

Reply:
yeah, just going to remove the argument